packages/experimental/Button/MIGRATION.md

```diff
@@ -0,0 +1,52 @@
+# Button Migration
+
+## STATUS: WIP 🚧
+
+This Migration guide is a work in progress and is not yet ready for use.
+
+## Migration from v0 Button
+
+### Component renames
+
+Common buttons now all map to `Button`:
+
+| v0 `Button`                             | v1 `Button`                                          |
+| --------------------------------------- | ---------------------------------------------------- |
+| `<Button text="Hello, world" />`        | `<Button>Hello, world</Button>`                      |
+| `<PrimaryButton text="Hello, world" />` | `<Button appearance='primary'>Hello, world</Button>` |
+| `<StealthButton text="Hello, world" />` | `<Button appearance='subtle'>Hello, world</Button>`  |
+
+### Props that remain as is
+
+- `children`
+- `disabled`
+- `icon`
+- `onClick`
+- `testID`
+- `tooltip`
+
+### Props no longer supported with an equivalent functionality in v1 Button
+
+- `componentRef` => Use regular `ref` instead
+- `content` => Pass the content as `children` instead
```

Comment:
Isn't content text, where children is a view hierarchy?

Reply:
Apparently web is making it so that content is now not a prop and you just pass in text as a child of the button instead... Not fully sure how that works so this might be a case where there's a difference due to technical limitation. @ValentinaKozlova was looking into this.

Reply:
Looking back through all the comments, I think this might just be a react vs react native difference, since web renders to HTML and presumably raw text is ok. It _might_ still with in react native but I would have thought we want a <Text> element. 
Point of discussion, not a review blocker. I'm interested to see what @ValentinaKozlova finds